velox/common/encode/Base64.cpp

```diff
@@ -22,6 +22,10 @@
 
 namespace facebook::velox::encoding {
 
+// Constants defining the size of binary and encoded blocks for Base64 encoding.
+constexpr static int kBinaryBlockSize = 3; // 3 bytes of binary = 24 bits
+constexpr static int kEncodedBlockSize = 4; // 4 bytes of encoded = 24 bits
```

Comment:
nit: 
> // 4 bytes of encoded = 24 bits

This comment is a bit confusing. How about we make it explicit in the variable name like:
 `kEncodedBlockByteSize`
similarly for `kBinaryBlockByteSize`

Reply:
Updated the code